src/nvim/ex_cmds.lua

```diff
@@ -483,8 +483,8 @@ return {
   },
   {
     command='chistory',
-    flags=bit.bor(TRLBAR),
-    addr_type=ADDR_LINES,
+    flags=bit.bor(RANGE, COUNT, TRLBAR),
+    addr_type=ADDR_OTHER,
```

Comment:
This should be `ADDR_UNSIGNED` which comes from patch v8.1.1261. Is this PR a short term fix for #13076 ?